src/script/azure_helpers.rb

```diff
@@ -58,13 +58,15 @@ def pull_request_commits(pull_request_id)
                 JSON.parse(response.body).fetch("value")
             end
 
-            def pull_request_auto_complete(pull_request_id, auto_complete_user_id, merge_strategy)
-                # https://docs.microsoft.com/en-us/rest/api/azure/devops/git/pull%20requests/update?view=azure-devops-rest-6.0
+            def pull_request_auto_complete(pull_request_id, pull_request_title, auto_complete_user_id, merge_strategy)
+                # https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-requests/update?view=azure-devops-rest-6.0
                 content = {
                     autoCompleteSetBy: {
                         id: auto_complete_user_id
                     },
                     completionOptions: {
+                        autoCompleteIgnoreConfigIds: [93], # ignore optional "work items must be linked" policy
```

Comment:
This ignore config does not match all uses. Other users may want other configs to be ignored while this remains. Further, if the policy is marked as optional, the PR will complete when all the required policies succeed.

If this has to be done, I recommend an opt in mechanism for which policies to be ignored provided as a config/variable. I wonder if it will be consistent when https://github.com/dependabot/dependabot-core/pull/4064 gets merged

Reply:
concerning [dependabot/dependabot-core#4064](https://github.com/dependabot/dependabot-core/pull/4064)
let's see when it gets merged, then we can decide what needs to be adapted.

Do you mean I shall provide an option in the pipeline task to provide an array of numbers?
Unfortunately, I am not really sure where the magic number comes from, I just saw it when manually configuring the PR in browser. But if I do not provide the number, the optional configured policy is required/blocking. 

I will update the PR here.

Reply:
@mburumaxwell please check. I added a new env variable `AZURE_AUTO_COMPLETE_IGNORE_CONFIG_IDS` to the docker image which can be configured via pipeline task input `autoCompleteIgnoreConfigIds`